packages/analytics/dashboard-renderer/src/components/layout/GridLayout.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="kong-ui-public-grid-layout">
+    <div
+      v-for="cell in totalCells"
+      :key="cell.key"
+      class="grid-cell"
+      :style="{
+        gridRow: `span ${cell.tile?.size.rows}`,
+        gridColumn: `span ${cell.tile?.size.cols}`,
```

Comment:
I'm not sure the optional chaining makes sense in the style attributes here; if they are undefined it will render invalid CSS